lib/clientv2/ippool.go

```diff
@@ -207,3 +213,64 @@ func (_ ipPools) validateAndSetDefaults(pool *apiv2.IPPool) error {
 
 	return nil
 }
+
+// maybeEnableIPIP turns enables globally IPIP if a default setting is not already configured
+// and the pool has IPIP enabled.
+func (c ipPools) maybeEnableIPIP(ctx context.Context, pool *apiv2.IPPool) error {
+	logCxt := log.WithField("IPIPMode", pool.Spec.IPIPMode)
+	if pool.Spec.IPIPMode == apiv2.IPIPModeNever {
+		logCxt.Debug("IPIP is not enabled for this pool - no need to check global setting")
```

Comment:
Does this really need the above context logger? We say right in the log that it's not enabled...

Reply:
Not strictly - but it does help verify the logic is correct